### ISSUE TITLE
[A11Y] Incorrect talkback output in StoryActivity

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#3898]
Incorrect talkback output in StoryActivity. Text repetition issue.

<img src="https://user-images.githubusercontent.com/9396084/136245117-62eadd88-6e92-4761-8a4c-fd701c4b4cef.png" width="250" />

**Comment by bkaur-bkj** [comment]
@rt4914 I would like to work on this issue  but I have a doubt on what should be the correct talkback output as we have mentioned the chapter name "What is a fraction" twice in text so it will obviously be repeated in talkback output

**Comment by rt4914** [comment]
@bkaur-bkj Yes you are correct. We can close this issue. Thanks a lot.
